util/test/checkChplInstall

```diff
@@ -104,6 +104,15 @@ log_info "Temporary test job directory: ${TMP_TEST_DIR}"
 function cleanup_tmp_dir()
 {
     log_debug "Removing ${TMP_TEST_DIR}"
+    local i
+    # sometimes on an NFS file system, rm needs to wait for the TEST_JOB exe file to close
+    for i in 1 2 3 4 5
```

Comment:
Got it. I'd prefer a `seq 5` here if this is something we're going to be tuning, but we can find out if this is something that we'll be tuning or not after a night of testing.